### PR TITLE
Adding built-in Recipes to database

### LINKED ISSUE #57: Generate some dummy recipes
Create a migration that will populate the database with dummy recipes, for testing and demonstration purposes.

### PR DESCRIPTION
**Pull request by efraimvis**
Created a data migration that will run automatically on database creation, and
populate it with some dummy recipes for testing and demonstration purposes.

This PR relies on #60, and should not be merged before it.
Also, this PR may not pass the pytest workflow at least until changes made in #64 are merged.
Closes #57.

### CODE REVIEW

**Comment by GilShechter** [comment]
You should add tests to make sure the data is saved into the database correctly.
Besides that, looking good. Nice work.

**Comment by natifridman** [comment]
> You should add tests to make sure the data is saved into the database correctly. Besides that, looking good. Nice work.

There is no need to add tests for migrations

**Review by GalOved — DISMISSED** [review]
Good work it is looking good

**Review by GilShechter — DISMISSED** [review]

File: tasties_app/migrations/0004_create_recipes.py [hunk 1]
```diff
@@ -0,0 +1,975 @@
...
+            if len(rating_sublist) > len(author_pool):
+                rating_sublist = ()
+            with transaction.atomic():
+                recipe = Recipe(
+                    author_id=author,
+                    title=title,
+                    description=description,
+                    directions=direction,
+                    minutes_to_make=minute,
+                    recipe_picture="static/images/" + picture,
```
Comment (GilShechter) [inline tasties_app/migrations/0004_create_recipes.py:937]: In order to present the images you will need to remove the `static/` at the beginning of the string.
I tested it locally.
When you perform `{% static 'img_url' %}` it already contains the static folder path.

Comment (efraimvis) [reply tasties_app/migrations/0004_create_recipes.py:937]: > In order to present the images you will need to remove the `static/` at the beginning of the string. I tested it locally. When you perform `{% static 'img_url' %}` it already contains the static folder path.

Thanks for the heads up. Will adjust.

**Review by GilShechter — DISMISSED** [review]
LGTM

**Review by GalOved — DISMISSED** [review]
Looking good nice work!.

**Review by natifridman — DISMISSED** [review]

**Review by ohadrevach — DISMISSED** [review]

**Comment by natifridman** [comment]
@efraimvis Please squash your commits

**Comment by efraimvis** [comment]
> @efraimvis Please squash your commits

@natifridman I will not be able to do that until late in the evening, I do not have access to my local repo at the moment. Would you consider merging this PR "as is"? I understand the need to comply with best practice, but in this particular case there's just one "junk commit".

**Comment by natifridman** [comment]
> @natifridman I will not be able to do that until late in the evening, I do not have access to my local repo at the moment. Would you consider merging this PR "as is"? I understand the need to comply with best practice, but in this particular case there's just one "junk commit".

We still need to follow our guidelines. As I see it there we have 2 options:

- Someone else from the team will fetch your PR and do the squash
- Squash the commit in the evening and make sure your team approves it, and we will merge it first thing in the morning

**Comment by efraimvis** [comment]
> @efraimvis Please squash your commits

@natifridman DONE.

**Review by ohadrevach — APPROVED** [review]

**Review by GalOved — APPROVED** [review]
Good work.

**Review by GilShechter — APPROVED** [review]

**Review by natifridman — APPROVED** [review]

**Pull request merged**